src/objects.js

```diff
@@ -1,41 +1,74 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name: name,
+          age: age}
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]){
+    return true;
+  } else {
+    return false;
+  }
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65){
+    return true;
+  } else {
+    return false;
+  }
 };
 
 const getAges = people => {
-  // your code here
+  let myArray = [];
```

Comment:
This can be a const as you are never reassigning the entire value of the variable myArray. You can mutate the contents of objects and arrays, even if they are declared as a const as the constant reference is a reference to the location the array is stored in memory, not to its contents.

Reply:
Great use of for loops. See if you can refactor using an iterator method from Wednesday